src/platforms/stlinkv3/platform.c

```diff
@@ -255,7 +255,15 @@ void platform_init(void)
 
 void platform_target_clk_output_enable(bool enable)
 {
-	(void)enable;
+	/* Regardless of swdptap.c, tri-state JTCK and JTMS */
```

Comment:
The `J` suffixes should not be here. They are ST anachronisms and the signals are just called `TCK` and `TMS`.

Reply:
Comment line edited, fixup applied, rebased to main. I guess I might want to tri-state TDI for JTAG as well? This is good enough for SWD.

Reply:
It's possible, though we've not heard of anyone having a problem with TDI not being tristated - for SWD-only rigs, it's not connected, and for JTAG it's a hard requirement w/ only pin-constrained micros really allowing it to be remapped to anything if they even support JTAG. That all said, we certainly wouldn't rule out the possibility.